types/jquery-lazyload/index.d.ts

```diff
@@ -15,7 +15,7 @@ declare namespace JQueryLazyLoad {
         container?: JQuery;
         data_attribute?: string;
         skip_invisible?: boolean;
-        appear?: null;
+        appear?: (elementsLeft: number, options: Options) => void | null;
```

Comment:
sorry again. but i do not think this is correct. this is a function that returns `void | null`.
I think what you want is:
```ts
appear?: ((elementsLeft: number, options: Options) => void) | null;
```

Reply:
Sorry, please check